src/scss/sections/_product-waitlist.scss

```diff
@@ -453,3 +453,32 @@
 		right: 8px;
 	}
 }
+
+.product-waitlist-form {
+	&-oos {
+		border-radius: 20px;
+
+		@include media-breakpoint-down(md) {
+			.input-group .form-control,
+			.input-group .btn {
+				border-radius: $border-radius;
+			}
+
+			.btn {
+				width: 100%;
+				margin-top: map-get($spacers, 1);
+			}
+		}
+
+		@include media-breakpoint-up(lg) {
+			.input-group {
+				background: $white;
+				border-radius: $border-radius-lg;
+			}
+			.form-control {
+				border: 0;
+				border-radius: $border-radius-lg;
```

Comment:
for border-radius use the default utility class directly on the DOM

Reply:
border radius for desktop and mobile is different, so cannot directly using the utility class